jmix-flowui/flowui/src/main/java/io/jmix/flowui/view/StandardDetailView.java

```diff
@@ -632,7 +632,12 @@ protected void setupEntityToEdit(T entityToEdit) {
     private boolean doNotReloadEditedEntity() {
         if (isEntityModifiedInParentContext()) {
             InstanceContainer<T> container = getEditedEntityContainer();
-            return getEntityStates().isLoadedWithFetchPlan(entityToEdit, container.getFetchPlan());
+            FetchPlan fetchPlan = container.getFetchPlan();
+            if (fetchPlan == null) {
```

Comment:
Probably we need to call `io.jmix.core.MetadataTools#isJpaEntity(io.jmix.core.metamodel.model.MetaClass)` to check if it's really DTO entity?

Reply:
Done